### ISSUE TITLE
performance bad for arctic dataset

### ISSUE THREAD

**Issue by ArtemisZGL** [NVIDIA/tacotron2#326]
Thanks for your great work firstly. I want to train the model in the cmu arctic cbl dataset, which is almost 1 hour for 1000 utterance. But after training for 500 epoch(about 40k step), the result is bad. If the text is in the training set, the audio can be understandable, but other texts result is bad. So I wonder how can I imporve the performance. Currently, my batch size is 12 for one gpu, and sample rate is set to 16kHz. The following is the alignment image:
![image](https://user-images.githubusercontent.com/31615877/78221450-f84fa480-74f5-11ea-8ba9-c83aeb8c8cde.png)

**Comment by rafaelvalle** [comment]
I'm assuming the model is overfitting on CMU Arctic CBL or stuck in a local minima giving that it is not using the attention properly.

Train the model with LibriTTS + CMU Arctic CBL or adjust the model parameters, e.g. decrease the PreNet number of dimensions.

In both cases do warm-start from the pre-trained model available on this repo.

**Comment by ArtemisZGL** [comment]
@rafaelvalle Thanks for your reply, I also have tried to use LJspeech to train, but I found the alignment image is bad after training 60 epoch with batch size 16. Should I train with more epoch or the batch size is the problem? The training loss and validation loss is about 0.4.
![image](https://user-images.githubusercontent.com/31615877/78378109-18b95500-7603-11ea-96ff-461dc905e0dc.png)


Further, I choose 10 male and female LibriTTS, each about 3 hour. And I train them from scratch. But the result seem so different. Can you give me some help? The following is the femal and male alignment.
![image](https://user-images.githubusercontent.com/31615877/78378562-b9a81000-7603-11ea-923a-067b66ee6f3f.png)
![image](https://user-images.githubusercontent.com/31615877/78378596-c298e180-7603-11ea-8d15-fd2d8a464373.png)

**Comment by ArtemisZGL** [comment]
@rafaelvalle hello, I notice that you have trained clb for tacotron2, can you share the haparms?(https://github.com/keithito/tacotron/issues/194) I have also train it with batch size 64 for 10000 epoch, but not good too.

**Comment by rafaelvalle** [comment]
Try changing **just** the prenet dim from 256 to 64.
